src/Bicep.Core/Semantics/ScopeResolution.cs

```diff
@@ -11,9 +11,9 @@ public enum ScopeResolution
     InheritAll,
 
     /// <summary>
-    ///   Inherit only function symbols from the parent scope.
+    ///   Inherit only function and variable symbols from the parent scope.
     /// </summary>
-    InheritFunctionsOnly,
+    InheritFunctionsAndVariablesOnly,
```

Comment:
Renamed this since in the places it's used, we are allowing inheritance of functions _and variables_.